src/dcos_e2e/node.py

```diff
@@ -338,10 +388,14 @@ def install_dcos_from_url(
             log_output_live=log_output_live,
             transport=transport,
             user=user,
+            sudo=True,
```

Comment:
Is there a test which covers this change?

Reply:
Yes, the new test_aws.py